### ISSUE TITLE
User is not able to drag the "AzureFun.js" grid section using single pointer mode of operation: A11y_Azure Tools for VS Code_Azure Function_Default Screen_DraggingMovement.

### ISSUE THREAD

**Issue by Harika1702** [microsoft/vscode-azurefunctions#4099]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.
 
### GitHubTags:
#A11yWCAG2.2; #A11yTCS; #A11ySev3; #Azure Tools for VS Code; #BM_AzureToolsforVSCode_MAC_April2024; #WCAG2.5.7; #Dragging Movements; #Benchmark; #FTP; #MAC; #Device;
 
### Environment and OS details:
Application Name: Azure Tools for VS Code
MAC Version: 14.4
 
### Repro Steps:
 
1. Open Visual studio code.
2. Click on extension button and search for "Azure Functions".
3. Click on install and move to the "Azure" button.
4. Observe that User is not able to resize "AzureFun.js" grid section using single pointer mode of operation.
 
### Actual Result:
User is not able to drag the "AzureFun.js" grid section using single pointer mode of operation
### Expected Result:
User should be able to drag the "AzureFun.js" grid section using single pointer mode of operation without dragging.
 
### User Impact:
Functionality provided only with dragging will create difficulty for users using input devices like a trackball, head pointer, eye-gaze system to perform the operation.
 
### Attachment:
 
![Azure Dragging moments1 mov](https://github.com/microsoft/vscode-azurefunctions/assets/100270074/f9b3966e-97ef-4ae8-87db-48eaa7c9d2a0)
![Dragging moments1 mov](https://github.com/microsoft/vscode-azurefunctions/assets/100270074/8ffd57f2-b856-4d96-9e19-049e47e73763)

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by MicroFish91** [comment]
Hi @Harika1702, I was wondering if this should actually be filed with [VS Code](https://github.com/microsoft/vscode/issues) instead.  It seems like this behavior would be present anytime you have an active sidebar open regardless of the individual extension installed (I also don't believe this level of change can be done on the extension level).  Could you please confirm?  Thank you!

**Comment by MicroFish91** [comment]
@Harika1702 @Yash14j Wanted to bump this dialogue, this seems to be more of a VS Code issue, would you be able to refile on their repository / close this one, thank you!
